Copyedit of .md files in repo. (#3)
thousandeyes/appd-integration-reference: snapshot-integration/readme.md

# ThousandEyes Snapshot Integration

> Create a ThousandEyes snapshot when an AppDynamics event or alert occurs.

Reviewer: ```suggestion
Create an AppDynamics HTTP Request Template that triggers a [Snapshot in ThousandEyes](https://docs.thousandeyes.com/product-documentation/tests/sharing-test-data). (Note that ThousandEyes Snapshots are different than AppDynamics snapshots).
```

> This uses AppDynamics' HTTP Request Template webhook integration feature, similar to the ThousandEyes alert webhook notification. 

Reviewer: I think we can remove this.

> For each ThousandEyes test associated with a given AppDynamics application (or tier or node), we will create a separate HTTP request to trigger a ThousandEyes snapshot via the ThousandEyes API. 

Reviewer: ```suggestion
Once created, our HTTP request template can be called by Actions defined in AppDynamics. We will define dynamic variables for the template - `testId`, `testName`, and `accountId` - that are specified by each Action, so that a single HTTP request template can be used to trigger a snapshot for any test in ThousandEyes. 
```

> We will create an HTTP request template so that the ThousandEyes snapshot time window and test ID can be provided dynamically.

Reviewer: ```suggestion
```

## Create HTTP Request Template

1. Under **Alert & Respond**, select **HTTP Request Template**.
2. Click **+** (New)
3. Set up a HTTP request template to trigger a ThousandEyes snapshot:

### Name

> 

Reviewer: ```suggestion
### Custom Templating Variables
Add the following custom templating variables:
- `accountId` - this is the ThousandEyes account that the snapshot will be created in. Default value can be set to a specific account ID, or specified by the calling Action.
- `testId` - the ID of the test in ThousandEyes to trigger a snapshot on. No default value - should be set by the calling Action. Note - a test's ID can be found by viewing a test in ThousandEyes and looking for the `testId` parameter in the URL. For example: 
 `https://app.thousandeyes.com/view/tests/?testId=1705574`
 - `testName` - the name of the test in ThousandEyes. No default value - should be set by the calling Action.
```

Name the HTTP request **ThousandEyes Snapshot**.

### Set the Request URL

Set the request URL to **POST https://api.thousandeyes.com/v6/snapshot.json**.

[httprequest-url.png]

### Set Authentication

Set authentication to **Basic**. Use your ThousandEyes username and API token as password.

### Set the Payload

Note - Using Apache VTL to format time string in request.

> In ThousandEyes, the test ID for the **adcapital** test is 1246117.

Reviewer: ```suggestion
```
